portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,55 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script>
```

Comment:
You can still keep the external JS script here.  I wouldn't recommend inline the JS since it will be cleaner to keep it separate.